CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
Shouldn't this a `Added` section?

Reply:
`defaultMenuIsOpen` option for `EXPERIMENTAL_Select`

Reply:
Yes, but it is a prop of the used library, so maybe it should be available... as not all props from the library are available, I'll use an added, makes more sense